server/api-manager.js

```diff
@@ -25,6 +25,11 @@ class MindbodyQueries {
   constructor() {
     this.requestNum = 0;
     this.authToken = null;
+    //cachedResults stores results that we've received from prior server requests
+    //it's a hash map, an array where the indexes are found by the calling the function
+    //mindbodyQueries.getHashCode(siteId + url). The value of each entry is a tuple of:
+    //{siteId: number, url: string, creationTime: timeInMs, result: Object}
```

Comment:
I included siteId as part of the cache key even though the Trello ticket didn't say to. Our app is mostly designed for a single site. Well, think about whether this causes any problems and needs to be changed/removed.